app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -296,3 +296,12 @@ $line-color-dark: #aaa;
 .name-input-group {
   width: 100%;
 }
+
+.glyphicon-star {
+  color: #428bca;
```

Comment:
Color literals like `#428bca` should only be used in variable declarations; they should be referred to via variable everywhere else.